model_compression_toolkit/exporter/fully_quantized/keras/builder/quantize_config_to_node.py

```diff
@@ -29,6 +29,12 @@
 from model_compression_toolkit.exporter.fully_quantized.keras.quantize_configs.weights_quantize_config import \
     WeightsQuantizeConfig
 
+SUPPORTED_QUANTIZATION_CONFIG = [WeightsQuantizeConfig,
```

Comment:
Not sure if this is the best location for this list, especially if it is not used in the context of this file.
Consider moving it to [fakely_quant_keras_exporter.py](https://github.com/sony/model_optimization/pull/376/files#diff-9fc31c2e54b68e28a9a9a580aaa65fb795d8a7b80db7538451c1095c2585a5a2) if it is only used there.